server/api.js

```diff
@@ -102,4 +102,27 @@ router.post(
 	}
 );
 
+
+
+
+
+router.get("/search", async (req, res) => {
+	try {
+ const searchQuery = req.query.searchQuery; // Get the search query from the request query parameters
+console.log(searchQuery)
```

Comment:
You can remove these debug statements - we don't want them on the production server